src/GitVersion.Core/Core/RepositoryStore.cs

```diff
@@ -166,7 +166,15 @@ public IEnumerable<IBranch> GetExcludedInheritBranches(Config configuration)
     public IEnumerable<IBranch> GetReleaseBranches(IEnumerable<KeyValuePair<string, BranchConfig>> releaseBranchConfig)
         => this.repository.Branches.Where(b => IsReleaseBranch(b, releaseBranchConfig));
 
-    public IEnumerable<IBranch> ExcludingBranches(IEnumerable<IBranch> branchesToExclude) => this.repository.Branches.ExcludeBranches(branchesToExclude);
+    public IEnumerable<IBranch> ExcludingBranches(IEnumerable<IBranch> branchesToExclude, bool excludeRemotes = false)
```

Comment:
Please specify the default parameter only on the interface level not on the implementation. It can be error prone and confusing when it will be changed in the interface but not here.

Reply:
I just want to point out that another public method was implemented in the same way, do you want me to still remove the default parameter on the implementation?

![image](https://user-images.githubusercontent.com/1633595/190136880-f47adcac-0202-4041-bfad-f0b9dc79195a.png)

Reply:
what do you think about @asbjornu ?

Reply:
I agree with @HHobeck. [Eric Lippert explains why here](https://ericlippert.com/2011/05/12/optional-argument-corner-cases-part-two/). Why the default parameter is on the implementation elsewhere must be an oversight and not intentional. We should move all default parameters to the interfaces where possible.

Reply:
done ✅